cloudformation/external-slurmdbd/external_slurmdbd/external_slurmdbd_stack.py

```diff
@@ -96,10 +172,16 @@ def _add_external_slurmdbd_load_balancer(
         self,
         target_group,
     ):
-        nlb = elbv2.NetworkLoadBalancer(self, "External-Slurmdbd-NLB", vpc=self.vpc, internet_facing=False)
+        nlb = elbv2.NetworkLoadBalancer(
+            self,
+            "External-Slurmdbd-NLB",
+            vpc=self.vpc,
+            vpc_subnets=ec2.SubnetSelection(subnets=[self.subnet]),
+            internet_facing=False,
+        )
 
         # add listener to NLB
-        listener = nlb.add_listener("External-Slurmdbd-Listener", port=22)
+        listener = nlb.add_listener("External-Slurmdbd-Listener", port=6819)
```

Comment:
Nitpick: Lets use a constant for the port as well, so that we dont have to change it at multiple places